core/domain/activity_jobs_one_off.py

```diff
@@ -471,3 +478,129 @@ def map(snapshot_model):
     def reduce(key, values):
         """Implements the reduce function for this job."""
         yield (key, len(values))
+
+
+class ValidateSnapshotMetadataModelsJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that validates whether each SnapshotMetadata model has a
+    corresponding CommitLog model pair and the corresponding parent model.
+    """
+
+    SnapshotMetadataModels = [
```

Comment:
SNAPSHOT_METADATA_MODELS, I think? (Similar to GET_HANDLER_ERROR_RETURN_TYPE)

Reply:
Done.